daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SExpr.scala

```diff
@@ -308,24 +310,27 @@ object SExpr {
         Array(),
         3,
         // case xs of
-        SECase(SEVar(1)) of (
-          // nil -> z
-          SCaseAlt(SCPNil, SEVar(2)),
-          // cons y ys ->
-          SCaseAlt(
-            SCPCons,
-            // f y (foldr f z ys)
-            SEVar(5)(
+        SECase(SEVar(1)) of (// nil -> z
```

Comment:
comment moved to previous line

Reply:
The auto formatter is doing that. I tried to undo it and the formatter did it again. :(